SKILL.tsv

```diff
@@ -4204,7 +4204,7 @@ SKILL_20150714_004203	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRat
 SKILL_20150714_004204	$Attribute Attack: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Number of Hits: #{CaptionRatio2}#{nl}Consumes 1 Magic Arrow
 SKILL_20150714_004205	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Consumes 1 Divine Arrow
 SKILL_20150714_004206	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Able to use while riding
-SKILL_20150714_004207	Fly circling around target area. Range defense of enemies within target area decreases.
+SKILL_20150714_004207	Hawk will encircle a specified location. Enemies within radius have their defence decreased.
```

Comment:
Enemies within radius have their defence decreased.-> Decreases the AoE defense of the target within the radius.